packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -3,21 +3,25 @@
 :root {
   --terra-consumer-body-background: url('https://wallpapercave.com/wp/5ynt7Gb.jpg');
```

Comment:
We should host this image on the HealtheLife S3 bucket and reference from there instead of wallpapercave.com

Reply:
@mhemesath @alex-bezek any thoughts on this?

Reply:
that isn't even the image we want I think...

Reply:
Varun, I think I'm ok with adding the image to S3...